vendor/k8s.io/kubernetes/pkg/api/validation/validation.go

```diff
@@ -3976,6 +3976,20 @@ func ValidateSecurityContextConstraints(scc *api.SecurityContextConstraints) fie
 			"required capabilities must be empty when all capabilities are allowed by a wildcard"))
 	}
 
+	if len(scc.Volumes) > 1 {
+		hasNone := false
+		for _, fsType := range scc.Volumes {
+			if fsType == api.FSTypeNone {
+				hasNone = true
+				break
+			}
+		}
+		if hasNone {
+			allErrs = append(allErrs, field.Invalid(field.NewPath("volumes"), scc.Volumes,
+				"if 'none' is specified, no other values are allowed"))
```

Comment:
If you inline this to the `if`, it will be possible to use `field.Index()` to complain on particular element.